<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="homeButton">
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usb1Button_1">
          <property name="text">
           <string>USB1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usb2Button_1">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>USB2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="homeButton_2">
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usb1Button_2">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>USB1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usb2Button_2">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>USB2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
